.rhcicd/clowdapp-recipients-resolver.yaml

```diff
@@ -191,6 +201,44 @@ objects:
           value: ${NOTIFICATIONS_KESSEL_RELATIONS_SECURE_CLIENTS}
         - name: KESSEL_DOMAIN
           value: ${KESSEL_DOMAIN}
+    initContainer:
+    - name: it-services-jks-builder
+      image: ${IT_SERVICES_JKS_BUILDER_IMAGE}:${IT_SERVICES_JKS_BUILDER_IMAGE_TAG}
+      command: ["bash", "/mnt/it-services-jks-builder/build-it-services-jks.sh"]
+      env:
+      - name: IT_SERVICES_KEYSTORE_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: it-services
+            key: it-services-keystorepassword
+            optional: true
```

Comment:
There's also the option to inherit all environment variables from the main container, but I only needed this one so I decided to make it explicit.